src/formatDate.js

```diff
@@ -8,7 +8,37 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splittedDate = date.split(fromFormat[3]);
+
+  const yearFormat = fromFormat.includes('YY') ? 'YY' : 'YYYY';
```

Comment:
It's more robust to check for the presence of 'YYYY' in the format instead of relying on the absence of 'YY' to assume 'YYYY'. This is because 'YY' is a substring of 'YYYY', and using 'includes' could lead to incorrect assumptions if the formats change in the future.